src/content/events/2024/bytesize_singularity_containers_HPC.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Bytesize: Containers for HPC with Singularity'
+subtitle: Marco Claudio De La Pierre, Seqera
+type: talk
+startDate: '2024-06-25'
+startTime: '13:00+02:00'
+endDate: '2024-06-25'
+endTime: '13:30+02:00'
+locations:
+  - links: https://kth-se.zoom.us/j/68390542812
+---
+
+This week Marco ([@marcodelapierre](https://github.com/marcodelapierre/)) will show us how to create singularity containers for high performance cluster (HPCs).
```

Comment:
will show us how to deploy software for Nextflow pipelines using Singularity containers, an optimal choice for high performance computing (HPC) clusters.

Reply:
```suggestion
This week Marco ([@marcodelapierre](https://github.com/marcodelapierre/)) will show us how to deploy software for Nextflow pipelines using Singularity containers, an optimal choice for high performance computing (HPC) clusters.
```